lesson_19/part_2/homework_19_2.py

```diff
@@ -0,0 +1,12 @@
+import requests
+
+files = {'image': open('example.jpg', 'rb')}
+upload_response = requests.post('http://127.0.0.1:8080/upload', files=files)
+print("Upload Response:", upload_response.json())
```

Comment:
It is better to add at least small validation. For ex if statuc_code == 200 print else print error or smth like that.